### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by dervondenbergen**
adds german translation (#2) and adds two additional translatable keys

maybe someone else should also take a look at the translations

### CODE REVIEW

File: public/locales/nl/translation.json [hunk 1]
```diff
@@ -27,10 +27,12 @@
     "liar-won": "De leugenaar, {{ liar }} is gewonnen! {{ liar }} heeft eigenlijk over {{ title}} gelezen.",
     "read-articles": "lees de artikels: ",
     "start-next": "start volgende ronde",
+    "score": "score",
```
Comment (Haroenv) [inline public/locales/nl/translation.json:30]:
```suggestion
    "scores": "scores",
```

File: src/App.js [hunk 2]
```diff
@@ -499,7 +499,7 @@ function TwoPlayerGame({ names, stop, language }) {
               </li>
             ))}
           </ul>
-          <p>scores:</p>
+          <p>{t('game.score')}:</p>
```
Comment (Haroenv) [inline src/App.js:502]:
```suggestion
          <p>{t('game.scores')}:</p>
```

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -27,10 +27,12 @@
     "liar-won": "The liar, {{ liar }} won! {{ liar }} actually read about {{ title }}",
     "read-articles": "read the articles: ",
     "start-next": "start next round",
+    "score": "score",
```
Comment (Haroenv) [inline public/locales/en/translation.json:30]:
```suggestion
    "scores": "scores",
```

File: public/locales/de/translation.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "preguessing": "Bitte reiche {{investigator}} das Gerät weiter.",
+    "start-guessing": "raten beginnen",
+    "guessing": "Die investigierende Person ({{investigator}}) ist jetzt am Zug. Du kannst Fragen zu diesen Artikeln stellen. Sobald du eine Vermutung hast welchen Artikel die andere Person gelesen hat, kannst du abstimmen!",
+    "scratchpad": "Hier hast du einen Schmierblock für deine Notizen:",
+    "scratchpad-tips": "Interessante details die sie wussten, haben sie das gerade gelesen? Oder ist das nur Zufall… Schreibe es zur Erinnerung auf.",
+    "investigator-won": "Die investigierende Person, {{ investigator }} hat gewonnen! {{ liar }} hat {{ title }} gelesen",
+    "liar-won": "Die lügende Person, {{ liar }} hat gewonnen! {{ liar }} hat eigentlich {{ title }} gelesen",
+    "read-articles": "Lies die Artikel: ",
+    "start-next": "nächste Runde starten",
+    "score": "Punkte",
```
Comment (Haroenv) [inline public/locales/de/translation.json:30]:
```suggestion
    "scores": "Punkten",
```

File: public/locales/de/translation.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{
+  "intro": {
+    "text": [
+      "In diesem Spiel gibt es zwei Rollen: die investigierende und die lügende Person. Jede Runde wechselt sich diese Rolle.",
```
Comment (Haroenv) [inline public/locales/de/translation.json:4]: Google Translate is suggesting `Ermittlungsperson`

File: public/locales/de/translation.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    ],
+    "language": "<0>sprache: </0>",
+    "player-one": "<0>spieler eins: </0>",
+    "player-two": "<0>spieler zwei: </0>",
+    "player-identical": "Spieler können nicht den gleichen Namen haben",
+    "start": "Start!",
+    "footer": "Dieses Spiel wurde inspiriert von <2>Two Of These People Are Lying</2> von Tom Scott und Matt Grey."
+  },
+  "game": {
+    "try-again": "try again",
```
Comment (Haroenv) [inline public/locales/de/translation.json:17]:
```suggestion
    "try-again": "versuchen Sie es nochmal",
```

or 

```suggestion
    "try-again": "versuche nochmal",
```

(not sure which fits best)

File: public/locales/de/translation.json [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    "choosing": "Die lügende Person ({{liar}}) ist jetzt am Zug. Entscheide dich für einen der Artikel, um dich über ein Gebiet zu informieren:",
+    "article": "Artikel {{number}}",
+    "done-reading": "Ich bin fertig",
+    "preguessing": "Bitte reiche {{investigator}} das Gerät weiter.",
+    "start-guessing": "raten beginnen",
+    "guessing": "Die investigierende Person ({{investigator}}) ist jetzt am Zug. Du kannst Fragen zu diesen Artikeln stellen. Sobald du eine Vermutung hast welchen Artikel die andere Person gelesen hat, kannst du abstimmen!",
+    "scratchpad": "Hier hast du einen Schmierblock für deine Notizen:",
+    "scratchpad-tips": "Interessante details die sie wussten, haben sie das gerade gelesen? Oder ist das nur Zufall… Schreibe es zur Erinnerung auf.",
+    "investigator-won": "Die investigierende Person, {{ investigator }} hat gewonnen! {{ liar }} hat {{ title }} gelesen",
+    "liar-won": "Die lügende Person, {{ liar }} hat gewonnen! {{ liar }} hat eigentlich {{ title }} gelesen",
```
Comment (Haroenv) [inline public/locales/de/translation.json:27]: is this Austrian? Google Translate tells me it's `liegende`

Comment (dervondenbergen) [reply public/locales/de/translation.json:27]: `liegen` is when you lie down.
`lügen` is when you lie like in this case.

File: public/locales/de/translation.json (hunk 5 above)
Comment (dervondenbergen) [reply public/locales/de/translation.json:4]: This makes no sense at all. Maybe `die ermittelnde Person` could work better.

Comment (Haroenv) [reply public/locales/de/translation.json:4]: I'm not native, so you can choose :)

Comment (dervondenbergen) [reply public/locales/de/translation.json:4]: I will change every instance of `investigierende` to `ermittelnde` which is a more fitting word.

**Pull request merged**